src/domobserver.js

```diff
@@ -82,6 +82,17 @@ export class DOMObserver {
     this.currentSelection.set(this.view.root.getSelection())
   }
 
+  ignoreSelectionMutation(sel) {
+    let desc = this.view.docView.nearestDesc(sel.anchorNode, true)
+
+    if (desc.node.isText) return false
```

Comment:
Why do you explicitly check for this? In any case, you probably shouldn't be dereferencing `desc` before checking whether it's null.

Reply:
Without this line selection changes in text nodes will always be ignored, presumably because of this in TextViewDesc:

```
ignoreMutation(mutation) {
    return mutation.type != "characterData"
}
```

Good point about the null check that needs to come first.

Reply:
Ah, right, we should probably change that in `TextViewDesc` rather than working around it here.